dm/master/election.go

```diff
@@ -61,6 +62,11 @@ func (s *Server) electionNotify(ctx context.Context) {
 					log.L().Error("scheduler do not started", zap.Error(err))
 				}
 
+				err = s.pessimist.Start(ctx, s.etcdClient)
+				if err != nil {
+					log.L().Error("pessimist do not started", zap.Error(err))
```

Comment:
if start pessimist failed, do we need close scheduler and skip set leader in L71?

Reply:
I think we need a mechanism to **transfer** _leader_ role to another DM-master member when any necessary components started fail later.

Reply:
and close the pessimist

Reply:
> I think we need a mechanism to **transfer** _leader_ role to another DM-master member when any necessary components (pessimist and scheduler in currenlty) started fail later.

ok, we can do this later

Reply:
@lichunzhu you can open a issue to trace `leader transfer support` later.

Reply:
OK